src/loaddb/load_server_loader.cpp

```diff
@@ -703,12 +703,12 @@ namespace cubload
 	m_error_handler.log_date_time_conversion_error (token, pr_type_name (attr.get_domain ().type->get_id ()));
       }
     else if (error_code == ER_OBJ_ATTRIBUTE_CANT_BE_NULL)
-           {
-              char class_attr[512];
+      {
+	char class_attr[512];
```

Comment:
It looks better to replace with a macro for the hard coded value 512.

Reply:
I will change it to "DB_MAX_IDENTIFIER_LENGTH * 2".